checker-qual/src/main/java/org/checkerframework/framework/qual/TargetLocations.java

```diff
@@ -7,31 +7,38 @@
 import java.lang.annotation.Target;
 
 /**
- * NOTE: This meta-annotation is <a
- * href="https://github.com/typetools/checker-framework/issues/1919"><b>not currently
- * enforced</b></a>.
- *
- * <p>A meta-annotation that restricts the type-use locations where a type qualifier may be written.
+ * A meta-annotation that restricts the type-use locations where a type qualifier may be applied.
  * When written together with {@code @Target({ElementType.TYPE_USE})}, the given type qualifier may
- * be written only at locations listed in the {@code @TargetLocations(...)} meta-annotation.
+ * be applied only at locations listed in the {@code @TargetLocations(...)} meta-annotation.
  * {@code @Target({ElementType.TYPE_USE})} together with no {@code @TargetLocations(...)} means that
- * the qualifier can be written on any type use.
+ * the qualifier can be applied on any type use. {@code @TargetLocations({})} will prevent the
+ * annotation from been applied in the source code, but you can achieve the same goal by writing
+ * {@code @Target({})}. So, we don't suggest writing the annotation with no type-use location
+ * supplied.
```

Comment:
```suggestion
 * the qualifier can be applied to any type use. {@code @TargetLocations({})} means that the
 * qualifier should not be used in source code. The same goal can be achieved by writing
 * {@code @Target({})}, which is enforced by javac itself. So, we do not suggest writing {@code @Target({})}.
```

Reflow...

Reply:
My comment is ambiguous, but what i prefer is writing `@Target({})` instead of `@TargetLocations({})`. That is enforced by javac, so i think it's better.

Reply:
Please discuss the distinction in this javadoc. `@Target({})` is enforced by javac and there is no way around it. `@TargetLocations({})` is enforced by the checkers and can be suppressed or ignored with a flag.